scripts/install-macos.sh

```diff
@@ -0,0 +1,268 @@
+            "armv6" | "armv7l")
+                ARCH=armv6l
+                ;;
+            "armv8")
+                ARCH=arm64
+                ;;
+            .*386.*)
+                ARCH=386
+                ;;
+            esac
+            PLATFORM="linux-$ARCH"
+        ;;
+        "Darwin")
+            PLATFORM="darwin-amd64"
+        ;;
+    esac
+
+    if [ -z "$PLATFORM" ]; then
+        echo "Your operating system is not supported by the script."
+        exit 1
+    fi
+
+    if [ '/bin/bash' == `which bash` ]; then 
+        shell_profile="$HOME/.bashrc"
+        echo "bash is installed"
+    else
+        echo "Kelp requires bash"
+    fi
+
+    {
+        echo '# GoLang'
+        echo "export GOROOT=${GOROOT}"
+        echo 'export PATH=$GOROOT/bin:$PATH'
+        echo "export GOPATH=$GOPATH"
+        echo 'export PATH=$GOPATH:$PATH'
+    } >> "$shell_profile"
+
+    if [ -d "$GOROOT" ]; then
+        echo "The Go install directory ($GOROOT) already exists."
+    fi
+
+    PACKAGE_NAME="go$VERSION.$PLATFORM.tar.gz"
+    TEMP_DIRECTORY=$(mktemp -d)
+
+    echo "Downloading $PACKAGE_NAME ..."
+    if hash wget 2>/dev/null; then
+        wget https://storage.googleapis.com/golang/$PACKAGE_NAME -O "$TEMP_DIRECTORY/go.tar.gz"
+    else
+        curl -o "$TEMP_DIRECTORY/go.tar.gz" https://storage.googleapis.com/golang/$PACKAGE_NAME
+    fi
+
+    if [ $? -ne 0 ]; then
+        echo "Download failed! Exiting."
+        exit 1
+    fi
+
+    echo "Extracting File..."
+
+    # sudo chown -R $USER: $HOME # https://github.com/golang/go/issues/27187
+    mkdir -p "$GOROOT"
+
+    tar -C "$GOROOT" --strip-components=1 -xzf "$TEMP_DIRECTORY/go.tar.gz"
+
+    # GOROOT=${GOROOT}
+    # PATH=$GOROOT/bin:$PATH
+    # GOPATH=$GOPATH
+    # PATH=$GOPATH:$PATH
+
+    mkdir -p "${GOPATH}/"{src,pkg,bin}
+    echo -e "\nGo $VERSION was installed into $GOROOT.\nMake sure to relogin into your shell or run:"
+    echo -e "\n\ restart script to update your environment variables."
+    echo "Tip: Opening a new terminal window usually just works. :)"
+    rm -f "$TEMP_DIRECTORY/go.tar.gz"
+}
+
+# Once we have Golang; finish the install processes inside the development directory to avoid errors (Glide)
+function cloneIntoDir() {
+    echo "Setting up Kelp folders in the Golang working directory"
+    
+    # check github.com/stellar/kelp
+    if [ -d "$GOPATH/github.com/stellar/kelp" ]; then
+        echo "Kelp dir exists, no need to clone."
+    else
+        # pwd
+        mkdir -p $GOPATH/github.com/stellar/kelp
+        echo "Cloning Kelp into $GOPATH/src/github.com/stellar/kelp"
+        git clone https://github.com/stellar/kelp.git $GOPATH/src/github.com/stellar/kelp
+    fi
+
+    cd $GOPATH/src/github.com/stellar/kelp
+}
+
+# After Golang install Glide
+function installGlide() {
+    echo "Installing Glide."
+    if curl --version; then
+        curl https://glide.sh/get | sh
+    elif wget --version; then
+        wget https://glide.sh/get | sh
```

Comment:
this won't work because `glide.sh` doesn't work anymore. we will need to find an alternative way to install glide, maybe using homebrew?